vignettes/namespaces.Rmd

```diff
@@ -0,0 +1,288 @@
+---
+title: "Hold Namespace For Me"
```

Comment:
It's a nice title but will it be clear enough?

Reply:
Good point! What about these:

 1. Why you should care about XML namespaces
 2. Understanding XML namespaces
 3. Fun with XML namespaces
 4. Using XML namespaces in {tinkr}
 3. Namespace: the final frontier
 4. A document by any other namespace would parse just as stylesheet
 
I will stop there because I have other bad puns, but I know this is not what we are aiming for (it is a serious problem for me). I think the title also depends on whether the solution I came up with in #48 works.

Reply:
```suggestion
title: "Understanding XML Namespaces"
```